spec/services/fixture_indexing_service_spec.rb

```diff
@@ -27,10 +26,41 @@
     expect(response["response"]["numFound"]).to eq 1
   end
 
-  it "can index a fake private file to Solr" do
-    FixtureIndexingService.index_to_solr(priv_oid)
-    solr = RSolr.connect url: "#{solr_url}/#{solr_core}"
-    response = solr.get 'select', params: { q: '*:*' }
-    expect(response["response"]["docs"].first.dig("id")).to eq("16189097-priv")
+  context "it can index private files" do
+    let(:priv_oid) { "16189097-priv" }
+
+    it "can determine if a file is private" do
```

Comment:
I don't think this is actually determining that the file is private, is it? It's just mapping the defined ID to the defined object.